app/templates/src/main/java/package/config/_DatabaseConfiguration.java

```diff
@@ -114,9 +116,9 @@ public SpringLiquibase liquibase(DataSource dataSource) {
         SpringLiquibase liquibase = new SpringLiquibase();
         liquibase.setDataSource(dataSource);
         liquibase.setChangeLog("classpath:config/liquibase/master.xml");
-        liquibase.setContexts("development, production");
+        liquibase.setContexts(liquiBasePropertyResolver.getProperty("context", "development, production"));
```

Comment:
I don't think anybody uses it, at least that wasn't documented, I wasn't using it, and I've never heard of people using it -> no problem to change it!
I'd rather have something clean, and I'll document it if needed